lc-hax/Scripts/Patches/ShotgunPatch.cs

```diff
@@ -0,0 +1,17 @@
+#pragma warning disable IDE1006
+
+using HarmonyLib;
+
+namespace Hax;
+
+[HarmonyPatch(typeof(ShotgunItem))]
+class ShotgunPatch {
+    [HarmonyPatch(nameof(ShotgunItem.ShootGun))]
+    static void Prefix() => SoundManager.Instance.timeSinceEarsStartedRinging = 0f;
+
+    [HarmonyPatch(nameof(ShotgunItem.Update))]
+    static void Prefix(ref float ___misfireTimer, ref bool ___hasHitGroundWithSafetyOff) {
```

Comment:
For better compatibility with other mods, we use `Postfix` if possible. In this case, there should be no harm.

```suggestion
    static void Postfix(ref float ___misfireTimer, ref bool ___hasHitGroundWithSafetyOff) {
```

Reply:
the misfire code happens inside `ShotgunItem.Update`, so its actually better to use `Prefix` for this particular case

Reply:
Yeap, but reading the `Update` loop. `misfireTimer` is only set if `misfireTimer <= 0.0f`. So in this case, `Postfix` should work.